components/p3a/p3a_metrics.h

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+#define BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+
+namespace brave::p3a {
```

Comment:
if everything else in the component is in `brave`, why introduce p3a?

Reply:
Previously it was in an anonymous namespace within brave_p3a_service.cc. Since `kCollectedHistograms` isn't very specific, I wanted additional scoping to make it clear this version is specific to the component.

The component itself has `BraveP3A` object name prefixes, so there's no confusion.